pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1095,19 +1101,22 @@ int lkm_seeker_proc_tail(struct pt_regs *ctx)
     if (!init_tailcall_program_data(&p, ctx))
         return -1;
 
-    int ret = check_is_proc_modules_hooked(&p);
-    if (ret != 0) {
+    int ret = find_modules_from_module_kset_list(&p);
+    if (ret == HID_MOD_RACE_CONDITION) {
+        tracee_log(ctx, BPF_LOG_LVL_WARN, BPF_LOG_ID_UNSPEC, ret);
```

Comment:
Maybe we should add a parser of the values in user-mode to the const name

Reply:
Don't think it's critical. What's more important IMO is tracee's supporting log with a message from eBPF... but that's not for this PR.

Reply:
> What's more important IMO is tracee's supporting log with a message from eBPF...

Ori, `tracee_log()` is not working? Did I get you right?

In fact, `BPF_LOG_ID_UNSPEC` won't help us at all if we use it for everything. When a new log message needs to be used, I suggest implementing it in ebpf and userland:

https://github.com/aquasecurity/tracee/blob/b8f55163665b01654022eaab32858088a28e6096/pkg/ebpf/c/types.h#L393-L407

https://github.com/aquasecurity/tracee/blob/b8f55163665b01654022eaab32858088a28e6096/pkg/ebpf/bpf_log.go#L18-L61

Reply:
tracee_log is working. What I meant is adding a string argument to tracee_log that will be passed to userspace as is.
Like a userspace logger... tracee_log("Hello from eBPF") for that matter.
With the current tracee_log, I'm using the return value to distinguish the type of error.

Reply:
> Like a userspace logger... tracee_log("Hello from eBPF") for that matter.

Doing that would increase a lot the logic of it. We've already faced inst limit exceeded before using it as it is (only the id of the log).

For now you can define ids in bpf and in userland letting the latter decode the log correctly, showing an static string (from errorMap). Just use your actual ret values as IDs.